SixTrack/sixtrack.s

```diff
@@ -3890,50 +3892,55 @@
           pi=4d0*atan(1d0)
 +ei
         crabfreq=ek(ix)*c1e3
-
+        crabamp=ed(ix)
+                                 
         do j=1,napx ! loop over particles
-         crabamp=(ed(ix)/ejfv(j))*c1e3                                   !hr03
 
+         kcrab=(((sigmv(j)/(clight*(e0f/e0)))*crabfreq)*2d0)*pi         &!hr03
```

Comment:
This is actually the crab phase for a given particle, not the wave number; thus the name is a bit off. I would suggest to pull the unchanging calculation out of the loop, then just keep the sigmv(j)/kcrab+phase inside the loop...

Reply:
True it is not a real wavenumber but not what I would mean with a phase in this context. It would be some kind of position dependent phase.. 

 Yeah that is sure possible to change.. Is it worth it?